drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -174,7 +174,7 @@ func (k *k8sSchedOps) ValidateVolumeSetup(vol *volume.Volume) error {
 
 		containerPaths := getContainerPVCMountMap(p)
 		for containerName, path := range containerPaths {
-			pxMountCheckRegex := regexp.MustCompile(fmt.Sprintf("^(/dev/pxd.+|pxfs.+) on %s.+", path))
+			pxMountCheckRegex := regexp.MustCompile(fmt.Sprintf("^(/dev/pxd.+|pxfs.*) on %s.+", path))
```

Comment:
Why change + to *. In a success scenario, there will always be characters after pxfs.

Reply:
On my cluster I observed that there was nothing after pxfs